config/initializers/sentry.rb

```diff
@@ -13,6 +13,14 @@ def filter_record_not_unique_exception_messages!(event, hint)
   end
 end
 
+def ignore_sidekiq_errors_if_retry(event)
+  job = event&.transaction&.context&.job
```

Comment:
`job` can be `nil` here (due to the safe navigation operator `&.`), so I think we should add a line below this:

```rb
return false if job.nil?
```

(false because we _don't_ want to ignore this event)

An alternative would be to use the safe navigation operator wherever we access `job`, but I think my suggestion above is more typical Ruby.